VirtoCommerce.Domain/Pricing/Model/Price.cs

```diff
@@ -18,8 +18,10 @@ public class Price : AuditableEntity, ICloneable
 		public decimal? Sale { get; set; }
 		public decimal List { get; set; }
 		public int MinQuantity { get; set; }
+	    public DateTime? StartDate { get; set; }
```

Comment:
And please add comments for these proeprties

Reply:
Done